osbs/cli/main.py

```diff
@@ -718,6 +720,10 @@ def cli():
                               help='isolated build')
     build_parser.add_argument('--arrangement-version', action='store', required=False,
                               help='version of inner template to use')
+    build_parser.add_argument('--signing-intent', action='store', required=False,
+                              help='signing intent of the ODCS composes')
```

Comment:
'override signing intent of ODCS composes'

Reply:
ok

Reply:
Not done yet.

Reply:
ahh missed, it fixing